index.html

```diff
@@ -85,4 +85,4 @@ <h2>Input tests <small>Note that you don't need to submit any form for data trac
         <a href="https://github.com/jbtronics/CrookedStyleSheets">Github Repo</a>
     </div>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Looks like editing in the browser on github.com adds a trailing newline 😕 sorry, that was unexpected